src/rules/function-url-no-scheme-relative/README.md

```diff
@@ -0,0 +1,35 @@
+# function-url-no-scheme-relative
+
+Disallow scheme relative urls.
+
+```css
+a { background-image: url('//www.google.com/file.jpg'); }
+/**                        ↑ 
+ *                  This scheme relative url */
+```
+
+This rule ignores variables inside `url` function (`$sass`, `@less`, `--custom-property`).
+
+## Options
+
+### `true`
+
+The following patterns are considered warnings:
+
+```css
```

Comment:
Your examples are in the wrong place. These three should be below and the one below should be here.